tycho-its/projects/resolver.split/org.eclipse.equinox.security/src/org/eclipse/equinox/internal/security/storage/SecurePreferencesRoot.java

```diff
@@ -196,7 +197,7 @@ else if (!response.booleanValue())
 		OutputStream stream = null;
 		try {
 			stream = StorageUtils.getOutputStream(location);
-			properties.store(stream, description);
+			ReproduciblePropertiesUtils.store(properties, stream, description);
```

Comment:
I don't think this makes much sense in a test-case....

Reply:
Sure, but it's easier to apply this fix everywhere to be sure.

Reply:
But this is a testcase that has nothing to do with tycho (and fails to compile because it has no dependency to tycho codebase at all)